test/rabit.jl

```diff
@@ -0,0 +1,10 @@
+function test_rabit()
```

Comment:
I personally think it is better to have the `@tests` here inside an `@testset`, instead of a `function`, and have plain `include` without `testset` in the main `runtests.jl`, but that is not a change for now.

Reply:
Yeah, I just retained the pattern used elsewhere in this repo for now.